plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -1137,6 +1138,7 @@ public boolean downloadToFile(String urlString, Plugin plugin, @CheckForNull Fil
 
             // both the download and zip validation passed
             if(success) {
+                System.out.println("Downloaded file for " + plugin.getName());
```

Comment:
I think this should only be enabled for verbose logging, it's a huge logging change otherwise
```suggestion
                logVerbose("Downloaded plugin " + plugin.getName());
```

Reply:
I was thinking about this myself and the reason why I decided not to use verbose is because there doesn't seem to be any positive confirmation that plugin was actually downloaded when I ran it.

There seems to be some logging here
https://github.com/jenkinsci/plugin-installation-manager-tool/blob/master/plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java#L1090-L1095
and here
https://github.com/jenkinsci/plugin-installation-manager-tool/blob/master/plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java#L1003-L1006
but it's not producing any output (perhaps another issue)

Reply:
it's quiet by default, and only output's if there's an error, you can pass `--list` or `--verbose` if you want to see output